cognite_toolkit/_cdf_tk/warnings.py

```diff
@@ -52,3 +64,40 @@ def __str__(self) -> str:
             for warning in group_list:
                 output.append(f"{'    ' * 2} * {warning!s}")
         return "\n".join(output)
+
+
+@dataclass(frozen=True)
+class GeneralWarning(ToolkitWarning):
+    severity: SeverityLevel | None = None
```

Comment:
```suggestion
    severity: ClassVar[SeverityLevel] 
```

Reply:
Like we discussed, lets keep severity as a property of the class itself and not the instances.